packages/angular/projects/ui-angular/src/lib/services/authenticator.service.ts

```diff
@@ -34,9 +34,9 @@ export class AuthenticatorService implements OnDestroy {
 
     const authService = interpret(machine).start();
 
-    this._machineSubscription = authService.subscribe((state) => {
-      this._authState = state;
-      this._facade = getServiceContextFacade(state);
+    this._machineSubscription = authService.subscribe((state: unknown) => {
+      this._authState = state as AuthMachineState;
+      this._facade = getServiceContextFacade(state as AuthMachineState);
```

Comment:
Why was this change made?

Reply:
I was getting an error that state is not the correct type for `_authState` in my IDE.